graphql/queries/getPreviousSubmissions.ts

```diff
@@ -23,6 +23,7 @@ const GET_PREVIOUS_SUBMISSIONS = gql`
         name
       }
       comments {
+        id
```

Comment:
I think its outside the scope of this PR, but just food for thought: I think c0d3 codebase should look into refactoring queries/mutations to use fragments. We could probably save a lot of duplication by refactoring to use fragments.

https://graphql.org/learn/queries/
https://www.apollographql.com/docs/react/data/fragments/

Reply:
thanks @anthonykhoa ! Great suggestion!

https://github.com/garageScript/c0d3-app/issues/1441